blockmanager.go

```diff
@@ -418,11 +475,8 @@ func (b *blockManager) DonePeer(sp *ServerPeer) {
 	}
 }
 
-// handleDonePeerMsg deals with peers that have signalled they are done.  It
-// removes the peer as a candidate for syncing and in the case where it was the
-// current sync peer, attempts to select a new best peer to sync from.  It is
-// invoked from the syncHandler goroutine.
-func (b *blockManager) handleDonePeerMsg(peers *list.List, sp *ServerPeer) {
+// removeDonePeerFromList removes the peer from the peers list.
+func (b *blockManager) removeDonePeerFromList(peers *list.List, sp *ServerPeer) {
```

Comment:
why is this on the blockManager?

Reply:
https://github.com/lightninglabs/neutrino/pull/282#discussion_r1366613433

Extrapolation of this:
https://github.com/lightninglabs/neutrino/blob/42a196facefac36282e68ddf4a02c9ce4601a0b0/blockmanager.go#L426-L434

Reply:
It contains no references to the blockManager in the function body and is therefore a spurious argument. Do not accept arguments you do not use.